addons/l10n_pe/data/account.account.template.csv

```diff
@@ -1,1229 +1,1089 @@
-chart7771,7771,Ganancia por medición de activos y pasivos financieros al valor razonable - Inversiones mantenidas para negociación,account.data_account_type_other_income,pe_chart_template,False
-chart7772,7772,Ganancia por medición de activos y pasivos financieros al valor razonable - Otras inversiones,account.data_account_type_other_income,pe_chart_template,False
-chart7773,7773,Ganancia por medición de activos y pasivos financieros al valor razonable - Otras,account.data_account_type_other_income,pe_chart_template,False
-chart7781,7781,Participación en resultados de entidades relacionadas - Participación en los resultados de subsidiarias y asociadas bajo el método del valor patrimonial,account.data_account_type_other_income,pe_chart_template,False
-chart7782,7782,Participación en resultados de entidades relacionadas - Ingresos por participaciones en negocios conjuntos,account.data_account_type_other_income,pe_chart_template,False
-chart7792,7792,Otros ingresos financieros - Ingresos financieros en medición a valor descontado,account.data_account_type_other_income,pe_chart_template,False
-chart781,781,Cargas cubiertas por provisiones,account.data_account_type_other_income,pe_chart_template,False
-chart791,791,Cargas imputables a cuentas de costos y gastos,account.data_account_type_revenue,pe_chart_template,False
-chart792,792,Gastos financieros imputables a cuentas de inventarios,account.data_account_type_revenue,pe_chart_template,False
-chart801,801,Margen comercial,account.data_account_type_equity,pe_chart_template,False
-chart811,811,Producción de bienes,account.data_account_type_equity,pe_chart_template,False
-chart812,812,Producción de servicios,account.data_account_type_equity,pe_chart_template,False
-chart813,813,Producción de activo inmovilizado,account.data_account_type_equity,pe_chart_template,False
-chart821,821,Valor agregado,account.data_account_type_equity,pe_chart_template,False
-chart831,831,Excedente bruto (insuficiencia bruta) de explotación,account.data_account_type_equity,pe_chart_template,False
-chart841,841,Resultado de explotación,account.data_account_type_equity,pe_chart_template,False
-chart851,851,Resultado antes del impuesto a las ganancias,account.data_account_type_equity,pe_chart_template,False
-chart881,881,Impuesto a las ganancias – Corriente,account.data_account_type_equity,pe_chart_template,False
-chart882,882,Impuesto a las ganancias – Diferido,account.data_account_type_equity,pe_chart_template,False
-chart891,891,Utilidad,account.data_account_type_equity,pe_chart_template,False
-chart892,892,Pérdida,account.data_account_type_equity,pe_chart_template,False
+id,code,name,user_type_id:id,chart_template_id:id,reconcile,tag_ids/id
+chart0111,0111,Bienes y valores entregados,account.data_account_off_sheet,pe_chart_template,False,
+chart0112,0112,Derechos sobre instrumentos financiero,account.data_account_off_sheet,pe_chart_template,False,
```

Comment:
Why are the same account names used multiple times? (i.e. `chart0112` and `chart0213`)

Reply:
One of the purpose of having the `po` files would be to have the sources in English